libexec/helpers

```diff
@@ -24,3 +24,27 @@ function curlw () {
 
   curl ${TLS_OPT} "$@"
 }
+
+function get_version_requested () {
+  if [ -z "${1}" ]; then
+    version_file="$(tfenv-version-file)"
+    if [ "${version_file}" != "${TFENV_ROOT}/version" ];then
+      version_requested="$(cat "${version_file}" || true)"
```

Comment:
Is it fair to say that the only change that specifically intends to fix the spaces issue is these quotes around the version file? This seems perfectly appropriate.

Reply:
You're right, quotation marks are the only important thing. 
I simply wanted to share the code between install and uninstall.